mergify_engine/config.py

```diff
@@ -359,6 +371,15 @@ def ApplicationAPIKeys(v: str) -> typing.Dict[str, ApplicationAPIKey]:
 if globals()["QUEUE_URL"] is None:
     QUEUE_URL = globals()["STORAGE_URL"]
 
+if globals()["TEAM_MEMBERS_CACHE_URL"] is None:
+    TEAM_MEMBERS_CACHE_URL = globals()["STORAGE_URL"]
+
+if globals()["TEAM_PERMISSIONS_CACHE_URL"] is None:
+    TEAM_PERMISSIONS_CACHE_URL = globals()["STORAGE_URL"]
+
+if globals()["USER_PERMISSIONS_CACHE_URL"] is None:
+    USER_PERMISSIONS_CACHE_URL = globals()["STORAGE_URL"]
```

Comment:
Do we want to stop doing this so we make sure to not make the same "mistake" again with onprem in the future?

Reply:
I proposed an alternative that should be compatible with onpremise. If it looks good to you, I will add some tests to ensure the migration will works well.